appveyor.yml

```diff
@@ -0,0 +1,64 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\phpspec
+
+environment:
+    matrix:
+        - PHP_DOWNLOAD_FILE: php-5.3.3-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.3.29-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.4.44-nts-Win32-VC9-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.5.29-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+        - PHP_DOWNLOAD_FILE: php-7.0-nts-windows-vc14-x86-r2e38918.zip
```

Comment:
IMO, you should not test against all PHP versions on appveyor: AppVeyor does not allow to run parallel jobs, which means that all jobs are running one after the other, making the build take a long time. We already cover multiple PHP versions on Travis, so we may not need to do the same on AppVeyor